docs/data/data-grid/aggregation/aggregation-next.md

```diff
@@ -0,0 +1,261 @@
+| `size` | Returns the amount of cells in the group                   | all                          |
+
+### Remove a built-in function
+
+#### Remove a built-in function for all columns
+
+You can remove some aggregation functions for all columns by passing a filtered object to the `aggregationFunctions` prop.
+
+In the example below, the `sum` aggregation function has been removed:
+
+{{"demo": "AggregationRemoveFunctionAllColumns.js", "bg": "inline"}}
+
+#### Remove a built-in function for one column
+
+You can limit the aggregation options in a given column by passing the `availableAggregationFunctions` property to the column definition.
+
+This lets you specify which options will be available, as shown below:
+
+```ts
+const column = {
+  field: 'year',
+  type: 'number',
+  availableAggregationFunctions: ['max', 'min'],
+};
+```
+
+In the example below, the **Year** column can be aggregated using the `max` and `min` functions, whereas all functions are available for the **Gross** column:
+
+{{"demo": "AggregationRemoveFunctionOneColumn.js", "bg": "inline"}}
+
+### Create custom functions
+
+You can pass custom aggregation functions to the `aggregationFunctions` prop.
+
+An aggregation function is an object with the following shape:
+
+```ts
+const firstAlphabeticalAggregation: GridAggregationFunction<string, string | null> =
+  {
+    // The `apply` method takes the values to aggregate and returns the aggregated value
+    apply: (params) => {
+      if (params.values.length === 0) {
+        return null;
+      }
+
+      const sortedValue = params.values.sort((a = '', b = '') => a.localeCompare(b));
+
+      return sortedValue[0];
+    },
+    // The `label` property defines the label displayed in the column header when this aggregation is being used.
+    label: 'firstAlphabetical',
+    // The `types` property defines which type of columns can use this aggregation function.
+    // Here, we only want to propose this aggregation function for `string` columns.
+    // If not defined, aggregation will be available for all column types.
+    columnTypes: ['string'],
+  };
+```
+
+In the example below, the grid has two additional custom aggregation functions for `string` columns—`firstAlphabetical` and `lastAlphabetical`:
+
+{{"demo": "AggregationCustomFunction.js", "bg": "inline", "defaultCodeOpen": false}}
+
+### Custom value formatter
+
+By default, the aggregated cell uses the value formatter of its column.
+But for some columns, the format of the aggregated value may have to differ from the format of the other cell values.
+
+You can provide a `valueFormatter` method to your aggregation function to override the column's default formatting.
+
+```ts
+const aggregationFunction: GridAggregationFunction = {
+  apply: () => {
+    /* */
+  },
+  valueFormatter: (params) => {
+    /* format the aggregated value */
+  },
+};
+```
+
+{{"demo": "AggregationValueFormatter.js", "bg": "inline", "defaultCodeOpen": false}}
```

Comment:
I've noticed this demo doesn't really use custom valueFormatter for aggregation function - is it intentional?

Reply:
True, it must have been lost along the way...
Fixed